jekyll/_docs/configuration.md

```diff
@@ -15,10 +15,6 @@ file, check out our [sample file]({{site.baseurl}}/config-sample/).
 
 Should you have a test failure, our [troubleshooting section]({{ site.baseurl }}/troubleshooting/)
 can likely tell you the best way to solve the problem.
-If you find yourself repeatedly consulting this guide, please
-[contact us](mailto:support@circleci.com) and let us know what you're working on.
-We'll try to make it easier for you.
```

Comment:
Do you think this is any different than the other ones? It strikes me as a bit more general, more a request for "let us know if there are rough spots," but I'm totally okay with axing it if it is also outdated.